packages/hint-css-prefix-order/package.json

```
@@ -0,0 +1,77 @@
+{
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "dependencies": {
+    "postcss": "^7.0.14"
+  },
+  "description": "Ensure vendor-prefixed versions of a CSS property are listed before the unprefixed version.",
+  "devDependencies": {
+    "@hint/parser-css": "^2.0.3",
+    "@hint/utils-tests-helpers": "^2.0.3",
+    "@types/debug": "0.0.31",
+    "@types/node": "8.0.14",
+    "ava": "^1.2.0",
+    "cpx": "^1.5.0",
+    "eslint": "^5.12.1",
+    "eslint-plugin-markdown": "^1.0.0",
+    "eslint-plugin-typescript": "^0.14.0",
+    "hint": "^4.3.1",
+    "markdownlint-cli": "^0.13.0",
+    "npm-link-check": "^3.0.0",
+    "npm-run-all": "^4.1.5",
+    "nyc": "^13.1.0",
+    "postcss": "^7.0.14",
+    "rimraf": "^2.6.3",
+    "typescript": "^3.3.1",
+    "typescript-eslint-parser": "^22.0.0"
```

This package is deprecated. It should be `` (or newer)
```suggestion
    "@typescript-eslint/parser": "1.2.0"
```

I don't know why we have it pinned though so maybe we should open an issue about that.